abracadabra/src/writer/hdf5_writer.cc

```diff
@@ -0,0 +1,40 @@
+#include "hdf5_writer.hh"
+
+#include <sstream>
+#include <cstring>
+#include <stdlib.h>
+#include <vector>
+
+#include <stdint.h>
+#include <iostream>
+
+
+hdf5_writer::hdf5_writer():
+  file_(0), irun_(0) {}
```

Comment:
I'm not sure I understand what are you referring to here...

Reply:
``` c++
myclass() : member(args) { body; }
myclass() : member{args} { body; }
```

The latter should be preferred to the former (look at what surrounds `args`), unless there is an overriding reason to the contrary, since *uniform initialization* was introduced in C++11.

Not desperately important, but developing the habit of using braces rather than parentheses when constructing, saves us time and pain, on average and in the long run.

Reply:
Done